app/controllers/webhook_controller.rb

```diff
@@ -24,38 +31,57 @@ def callback
 
     events = client.parse_events_from(body)
     events.each do |event|
-      logger.debug "***********************************************************"
+      talk_id = event["source"]["groupId"]
+      talk_id ||= event["source"]["roomId"]
+      talk_id ||= event["source"]["userId"]
```

Comment:
ここは一回一回代入しなくても `||` をチェーンして使えたりします

```suggestion
      source = event["source"]
      talk_id = source["groupId"] || source["roomId"] || source["userId"]
```

Reply:
これも便利ですね！やってみます